src/Illuminate/Bus/Queueable.php

```diff
@@ -96,6 +96,8 @@ public function dispatchNextJobInChain()
         if (! empty($this->chained)) {
             dispatch(tap(unserialize(array_shift($this->chained)), function ($next) {
```

Comment:
Can you do ` function (Queueable $next) {` ?
This allows for better code inspection 👍